osu.Game/Online/Rooms/PlaylistItem.cs

```diff
@@ -113,7 +113,7 @@ public void MapObjects(IRulesetStore rulesets)
         }
 
         public bool ShouldSerializeID() => false;
-        public bool ShouldSerializeapiBeatmap() => false;
+        public bool ShouldSerializeAPIBeatmap() => false;
```

Comment:
Not confident about this one. This is a newtonsoft "convention" property that relies on being named the same as the `apiBeatmap` field. Need to doublecheck that this hasn't regressed.

Reply:
Missed that one. these methods need explicit comments in them.

Reply:
Yeah I'll get a branch going to revert this and also mark the other cases of this more clearly (not too many of these, thankfully).